JSTests/stress/import-lhs.js

```diff
@@ -0,0 +1 @@
+import ( "file" + ".js" ) [ 0 ] = 0 ;
```

Comment:
Could you remove the spaces here to make it clear what's happening? I thought this was an `import` _statement_ and I was confused for a solid five minutes. 😅